OpenOversight/app/main/views.py

```diff
@@ -297,6 +298,9 @@ def submit_data():
 
                  flash('File {} successfully uploaded!'.format(original_filename))
             except:
+                current_app.logger.error('Error uploading to S3: {}'.format(
+                    sys.exc_info()
```

Comment:
do we just want to to_string this tuple? I'm fine if so, I just want error logging :)